.github/workflows/release_prep.sh

```
@@ -14,9 +14,63 @@ ARCHIVE="rules_quarkus-$TAG.tar.gz"
+  sha=$(_sha256 "$released")
+  SHA_BLOCK+=$'\n'"    \"${minor}\": \"${sha}\","
+  CHECKSUM_ROWS+="| \`${released}\` | \`${sha}\` |"$'\n'
+done
+SHA_BLOCK+=$'\n'"}"
+
+# ---- Source archive with checksums patched in -------------------------------
 # NB: configuration for 'git archive' is in /.gitattributes
-git archive --format=tar --prefix=${PREFIX}/ ${TAG} | gzip > $ARCHIVE
-INTEGRITY=$(openssl dgst -sha256 -binary "$ARCHIVE" | openssl base64 -A)
+workdir=$(mktemp -d)
+git archive --format=tar --prefix=${PREFIX}/ ${TAG} | tar -C "$workdir" -xf -
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)